### DISCUSSION TITLE
Tutorial 4: Post your HPL Scores

### DISCUSSION THREAD

**Discussion by nyameko** [chpc-tech-eval/scc#114] [Show and tell]
Post and image of your HPL Scores here.

**Comment by ndumezulu-ngwenya** [comment]
UNISA-SCT'24

HPL Scores
![xhpl](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/89075456/e46dd1ef-c957-42d3-a358-2faaed288991)

**Comment by Talhahpatelia** [comment]
## Wits-A 

<img width="960" alt="push_ups" src="https://github.com/user-attachments/assets/c212e284-1422-4cc3-8183-27d2b6065733">

**Comment by Kohzmik** [comment]
CUT_Team
![newplot(1)](https://github.com/user-attachments/assets/5f589d10-896c-4327-bb1f-34585f09d977)

**Comment by hoperiley0806** [comment]
**Parallel Pioneers**

![Screenshot (137)](https://github.com/user-attachments/assets/6f681c6a-9123-4aeb-95bf-78e5b9ce6f5a)

**Comment by SfisoMncube** [comment]
**Khulisa IP-A**

![image](https://github.com/user-attachments/assets/c413e355-2963-43dd-ac5f-c89978417e7a)

**Comment by LesegoSenamela** [comment]
Algorithm Avengers
![newplot (1)](https://github.com/user-attachments/assets/7aae8a98-b3c6-44fe-9749-3e2b7dad3684)

**Comment by AnnaLukose** [comment]
**EEElite (UCT)**
![hpl](https://github.com/user-attachments/assets/2f4e0828-b4d4-4b27-9e72-510166a4e208)

**Comment by huziibee** [comment]
Wits-i
![newplot](https://github.com/user-attachments/assets/63631175-10f6-450d-a15d-ed9641731ae3)

**Comment by MphephuSamuel** [comment]
computer champions

<img width="1919" height="1079" alt="Screenshot 2025-07-10 001809" src="https://github.com/user-attachments/assets/8942effd-d5e0-4e11-88ac-cfe435bf1734" />

**Comment by Cloud-Strive** [comment]
Runtime Blinders

<img width="763" height="360" alt="Screenshot (20)" src="https://github.com/user-attachments/assets/1881a68d-6c1a-408d-bb24-e4751c99d47a" />

**Comment by ShavirV** [comment]
tuksbyters
<img width="984" height="525" alt="node nums" src="https://github.com/user-attachments/assets/c1b0ae09-fe3c-4626-ad91-ea86f6f68f56" />

**Comment by DCspearsv2** [comment]
DC_Spears
<img width="1300" height="762" alt="Screenshot 2025-07-11 190742" src="https://github.com/user-attachments/assets/905bd6e3-42dd-4060-90c3-9d858e4692f0" />

**Comment by ntando-memela** [comment]
Wits1
<img width="1261" height="360" alt="newplot" src="https://github.com/user-attachments/assets/7f3f5af9-414c-4947-b77c-93042798b806" />
<img width="1860" height="969" alt="Screenshot from 2025-07-12 16-49-59" src="https://github.com/user-attachments/assets/14173f0f-ac93-4ca5-b8e9-27f99e7d07d2" />
<img width="1860" height="969" alt="Screenshot from 2025-07-12 16-06-59" src="https://github.com/user-attachments/assets/68b49c87-88e8-446b-878b-ad6e9e0afd48" />

**Comment by Iloke-Alusala** [comment]
0V1RCL0CK1D
<img width="1008" height="360" alt="newplot" src="https://github.com/user-attachments/assets/37001064-f85c-4a50-b126-51b4bf77944a" />

**Comment by Kevin-Ebrahim** [comment]
Wits-A
<img width="1523" height="360" alt="BENCHMARKS" src="https://github.com/user-attachments/assets/58088341-acd2-4928-b58f-ba9a2964b252" />
<img width="1920" height="1080" alt="1HEADNODE" src="https://github.com/user-attachments/assets/1cd89a35-85f6-4a9f-ab25-a20852383734" />
<img width="1920" height="1080" alt="2OPENBLASREPO" src="https://github.com/user-attachments/assets/c64d854f-80c5-41cd-9324-d4884b699f71" />
<img width="1920" height="1080" alt="3OPENBLASCOMPILED" src="https://github.com/user-attachments/assets/b3ff1cb3-77f0-4317-8ae7-7754edee025e" />
<img width="1920" height="1080" alt="4INTEL1NODE" src="https://github.com/user-attachments/assets/7ac250bf-99a5-44cd-9338-3098319768b3" />
<img width="1920" height="1080" alt="5INTEL2NODE" src="https://github.com/user-attachments/assets/401a8b13-d08d-4235-9edf-9dbf59076d16" />
<img width="1920" height="1080" alt="6INTEL3NODE" src="https://github.com/user-attachments/assets/922e1842-b5f4-42b8-af37-51da769c5309" />

**Comment by JayJay-lab** [comment]
Sol_Plaatje_A  

<img width="1081" height="360" alt="newplot" src="https://github.com/user-attachments/assets/bf8b835c-53c4-4b6e-b709-0ec823887d29" />

**Comment by CadmusKei** [comment]
Jargon (UWC) | SCC 2026 | HPL Scores Jupyiter
<img width="1460" height="361" alt="image" src="https://github.com/user-attachments/assets/384b7249-489e-4752-840d-984a9b5c05e4" />

**Comment by unisaqcqueens641-hue** [comment]
UNISA_QC_QUEENS<img width="1600" height="901" alt="tut 4 no 3 part 1(1)" src="https://github.com/user-attachments/assets/1905fbb6-40d9-4316-b1c1-db633f5efa21" />
HPLScores

**Comment by VerilM** [comment]
Wits-1: HPL Scores

<img width="1422" height="360" alt="newplot" src="https://github.com/user-attachments/assets/7cea7bbc-6869-4412-b871-927cc24af963" />

**Comment by Cleve0602** [comment]
### DC Axes(CPUT) SCC26: HPL JUPYTERLAB ### 
<img width="965" height="375" alt="newplot" src="https://github.com/user-attachments/assets/5c7e2da0-68f7-4fa2-b9b2-9207fa15f78f" />
<img width="1383" height="870" alt="Python Jupiter" src="https://github.com/user-attachments/assets/ca347f3d-1976-4506-b218-ea2d2c3964e0" />

**Comment by deexglitch** [comment]
<img width="296" height="113" alt="image" src="https://github.com/user-attachments/assets/4a1ad53b-064f-4fcf-b7c2-5521fd3dadc9" />
<img width="1570" height="697" alt="Screenshot 2026-07-10 153410" src="https://github.com/user-attachments/assets/9840b327-1ced-4d82-94c7-f6f20854426c" />
<img width="743" height="341" alt="image" src="https://github.com/user-attachments/assets/510f7fac-05d3-4369-96d0-fde617713743" />

UPtime

**Comment by ThapeloVundla** [comment]
DC Spears

<img width="1372" height="587" alt="image" src="https://github.com/user-attachments/assets/25f881b6-8fde-4ae4-a306-55d88ba3b4be" />

**Comment by Lihle0512** [comment]
ChaosCompilers

<img width="866" height="765" alt="Screenshot from 2026-07-10 22-52-17" src="https://github.com/user-attachments/assets/dbf8a8f4-d838-4622-919b-d0938bd32bf6" />

**Comment by tDesh503** [comment]
Resilient Geeks
<img width="1397" height="505" alt="image" src="https://github.com/user-attachments/assets/2a128524-6aa5-473e-82e0-0d79ca67b592" />
